samples/matter/light_bulb/sysbuild/multiprotocol_rpmsg/prj.conf

```diff
@@ -0,0 +1,45 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+## Disable serial and UART interface.
+CONFIG_SERIAL=n
+CONFIG_UART_CONSOLE=n
```

Comment:
Generic comment to the file, is it going to work with all kinds of prj+suffix.confs? E.g. for child images we had to duplicate it in case of having several different prj config names.

Reply:
Not supported, there is a plan for this but it is a long way off

Reply:
Looks like there is a chance:
https://github.com/nrfconnect/sdk-nrf/tree/main/samples/zigbee/light_switch/sysbuild/multiprotocol_rpmsg
Again, @nordicjm  can you please comment on that? :)

Reply:
Comment on what, what is the question? If you have an application and configure it to use prj_my_config.conf, it will apply to that image only, the rest of the images will use prj.conf as normal vs in child/parent image they would all use prj_my_config.conf

Reply:
Currently whenever we want a dedicated application configuration, for instance prj_release.conf, we need to provide the file with the exact same name in the `child_image/image_name` location even if prj_release.conf is the same as the default prj.conf for `image_name`. For example:
https://github.com/nrfconnect/sdk-nrf/blob/main/samples/matter/lock/prj_release.conf and
https://github.com/nrfconnect/sdk-nrf/blob/main/samples/matter/lock/child_image/multiprotocol_rpmsg/prj_release.conf

The question is: does it work the same in sysbuild (sysbuild/multiprotocol_rpmsg/*)?

Reply:
> Not supported, there is a plan for this but it is a long way off